android/BOINC/app/src/main/java/edu/berkeley/boinc/client/Monitor.java

```diff
@@ -1180,7 +1180,12 @@ public int addAcctMgrErrorNum(String url, String userName, String pwd)
                 throws RemoteException {
             AcctMgrRPCReply acctMgr = clientInterface.addAcctMgr(url, userName, pwd);
             if (acctMgr != null) {
-                return acctMgr.error_num;
+                if(acctMgr.messages.get(0).equals("User not found or password wrong.")) {
```

Comment:
Unfortunately, this will not work either: as I said previously, we don't know what message will be sent from the server.
According to my observations, there is next situation on wrong credentials (random user/email + random password).
- BAM! - No error code sent
![BAM](https://user-images.githubusercontent.com/3234209/67079169-fafe4d80-f192-11e9-8ab2-5a8ad4fbb9b7.PNG)
- GRCPool - No error code sent
![GRCPool](https://user-images.githubusercontent.com/3234209/67079205-0cdff080-f193-11e9-9de8-5a7bea118de1.PNG)
- Grid Republic - Error code sent
![GridRepublic](https://user-images.githubusercontent.com/3234209/67079237-1bc6a300-f193-11e9-85fb-fe63aec558a6.PNG)
- Science United - Error code sent
![ScienceUnited](https://user-images.githubusercontent.com/3234209/67079247-25500b00-f193-11e9-882d-e56eaeadba8a.PNG)

So having this information next algorythm should be used:
- if there is an error code different from 0 - Show error message from response
- if there is an error code 0 and message from response is not empty - Show error message from response
- if there is an error code 0 and message from response is empty - Consider that everything went fine and proceed